### PR TITLE
Minor fixes to migration docs

### PR DESCRIPTION
**Pull request by FaithOmbongi**

### CODE REVIEW

**Comment by Lauragra** [comment]
@FaithOmbongi , do you want to restore the overview topic per the feedback from Angela? Thanks.

**Comment by FaithOmbongi** [comment]
@Lauragra , yes, will do.

File: api-reference/beta/toc.yml [hunk 1]
```diff
@@ -92,7 +92,7 @@ items:
       displayname: Migration, Azure AD Graph
       items:
       - name: Overview
-        href: /graph/migrate-azure-ad-graph-planning-checklist?context=graph%252fapi%252fbeta
+        href: /graph/migrate-azure-ad-graph-overview?context=graph%252fapi%252fbeta
```
Comment (Lauragra) [inline api-reference/beta/toc.yml:95]: When you click on this topic in the TOC, it shows under the checklist just under step 1. The topic should come before the checklist. Can you please investigate why it's doing that?

Comment (FaithOmbongi) [reply api-reference/beta/toc.yml:95]: Hmm, just seen this. Let me investigate.

**Comment by jasonjoh** [comment]
Docs Build status updates of commit _[7cbd3f6](https://github.com/microsoftgraph/microsoft-graph-docs/commits/7cbd3f6846613aefb57dc6f89995a4508715b7a8)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/toc.yml](https://github.com/microsoftgraph/microsoft-graph-docs/blob/AADGraphMigrationFixes/api-reference/beta/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/accesspackageassignment-filterbycurrentuser?view=graph-rest-beta&branch=pr-en-us-13927) (graph-rest-beta) |
[api-reference/v1.0/toc.yml](https://github.com/microsoftgraph/microsoft-graph-docs/blob/AADGraphMigrationFixes/api-reference/v1.0/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/1.0.json?view=graph-rest-1.0&branch=pr-en-us-13927) (>= graph-rest-1.0) |
[concepts/migrate-azure-ad-graph-app-registration.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/AADGraphMigrationFixes/concepts/migrate-azure-ad-graph-app-registration.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/migrate-azure-ad-graph-app-registration?branch=pr-en-us-13927) |
[concepts/migrate-azure-ad-graph-audit-api-use.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/AADGraphMigrationFixes/concepts/migrate-azure-ad-graph-audit-api-use.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/migrate-azure-ad-graph-audit-api-use?branch=pr-en-us-13927) |
[concepts/migrate-azure-ad-graph-authentication-library.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/AADGraphMigrationFixes/concepts/migrate-azure-ad-graph-authentication-library.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/migrate-azure-ad-graph-authentication-library?branch=pr-en-us-13927) |
[concepts/migrate-azure-ad-graph-client-libraries.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/AADGraphMigrationFixes/concepts/migrate-azure-ad-graph-client-libraries.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/migrate-azure-ad-graph-client-libraries?branch=pr-en-us-13927) |
[concepts/migrate-azure-ad-graph-deploy-test-extend.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/AADGraphMigrationFixes/concepts/migrate-azure-ad-graph-deploy-test-extend.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/migrate-azure-ad-graph-deploy-test-extend?branch=pr-en-us-13927) |
[concepts/migrate-azure-ad-graph-feature-differences.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/AADGraphMigrationFixes/concepts/migrate-azure-ad-graph-feature-differences.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/migrate-azure-ad-graph-feature-differences?branch=pr-en-us-13927) |
[concepts/migrate-azure-ad-graph-method-differences.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/AADGraphMigrationFixes/concepts/migrate-azure-ad-graph-method-differences.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/migrate-azure-ad-graph-method-differences?branch=pr-en-us-13927) |
[concepts/migrate-azure-ad-graph-overview.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/AADGraphMigrationFixes/concepts/migrate-azure-ad-graph-overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/migrate-azure-ad-graph-overview?branch=pr-en-us-13927) |
[concepts/migrate-azure-ad-graph-planning-checklist.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/AADGraphMigrationFixes/concepts/migrate-azure-ad-graph-planning-checklist.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/migrate-azure-ad-graph-planning-checklist?branch=pr-en-us-13927) |
[concepts/migrate-azure-ad-graph-property-differences.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/AADGraphMigrationFixes/concepts/migrate-azure-ad-graph-property-differences.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/migrate-azure-ad-graph-property-differences?branch=pr-en-us-13927) |
[concepts/migrate-azure-ad-graph-request-differences.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/AADGraphMigrationFixes/concepts/migrate-azure-ad-graph-request-differences.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/migrate-azure-ad-graph-request-differences?branch=pr-en-us-13927) |
[concepts/migrate-azure-ad-graph-resource-differences.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/AADGraphMigrationFixes/concepts/migrate-azure-ad-graph-resource-differences.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/migrate-azure-ad-graph-resource-differences?branch=pr-en-us-13927) |
[redirects/.openpublishing.redirection.0920.json](https://github.com/microsoftgraph/microsoft-graph-docs/blob/AADGraphMigrationFixes/redirects/.openpublishing.redirection.0920.json) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C9%5C3%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C202109031543485291-13927%5Cworkflow_report.html?sv=2018-03-28&sr=b&sig=I%2FBRO0UBJhdX9cpJWudIOdTJhkqBFUmaHryvKGhMrYA%3D&st=2021-09-03T15%3A40%3A25Z&se=2021-10-04T15%3A45%3A25Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/77e68b91-cf14-3558-43ca-0d6d735c492d?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by FaithOmbongi** [comment]
Bug refused to resolve itself. With the help of SRE, decided to copy my commits over to a new branch #14146.

cc @Lauragra

**Comment by FaithOmbongi** [comment]
The changes are live. Removing the changes requested label to avoid unactionable notifications.

**Pull request closed without merging**